src/debugger/direct/directDebugSession.ts

```diff
@@ -53,22 +53,24 @@ export class DirectDebugSession extends DebugSessionBase {
                         return TelemetryHelper.generate("launch", extProps, (generator) => {
                             return this.appLauncher.launch(launchArgs)
                                 .then(() => {
-                                    return this.appLauncher.getPackagerPort(launchArgs.cwd);
-                                })
-                                .then((packagerPort: number) => {
-                                    launchArgs.port = launchArgs.port || packagerPort;
-                                    this.attachRequest(response, launchArgs).then(() => {
+                                    if (launchArgs.debuggingEnabled) {
+                                        launchArgs.port = launchArgs.port || this.appLauncher.getPackagerPort(launchArgs.cwd);
+                                        this.attachRequest(response, launchArgs).then(() => {
+                                            resolve();
+                                        }).catch((e) => reject(e));
```

Comment:
Should we show the similar error like here? https://github.com/microsoft/vscode-react-native/pull/1319/files#diff-28e7cf10d05219e87adfabc5cb1c1c93R66-R69 
Also I think that should be localized.